src/ui/Analytics/AnalyticsActionListMeta.ts

```diff
@@ -202,99 +202,284 @@ export interface IAnalyticsResultsLayoutChange {
   resultsLayoutChangeTo: string;
 }
 
+/**
+ *
```

Comment:
Should this have a comment ? Does it still show up in the documentation even if it is "empty" ?